apps/datagouvfr/lib/datagouvfr/client/api.ex

```diff
@@ -6,21 +6,22 @@ defmodule Datagouvfr.Client.API do
 
   @type response :: {:ok, any} | {:error, any}
 
+  @spec post_process({:ok, %HTTPoison.Response{body: binary()}}) :: {:ok, map()} | {:error, any()}
   def post_process({:ok, %HTTPoison.Response{body: body} = response}) when is_binary(body) do
     case Jason.decode(body) do
       {:ok, body} -> post_process({:ok, %{body: body, status_code: response.status_code}})
-      error -> post_process(error)
+      {:error, error} -> post_process({:error, error})
```

Comment:
whou you found an error with the dializer?

Reply:
I just read the documentation https://hexdocs.pm/jason/Jason.html#decode/2